sdk/client/Cargo.toml

```diff
@@ -23,6 +23,7 @@ hex = "0.4.3"
 serde = { version = "1.0.124", features = ["derive"] }
 serde_json = "1.0.64"
 tracing = "0.1.26"
+void = "1.0.2"
```

Comment:
Remove this dependency and use [`Infallible`](https://doc.rust-lang.org/stable/std/convert/enum.Infallible.html) instead maybe?

Reply:
Oh wow, perfect. didn't even realize that existed